net/acme-acmesh/files/hook.sh

```diff
@@ -0,0 +1,128 @@
+#!/bin/sh
+set -u
+ACME=/usr/lib/acme/acme.sh
+LOG_TAG=acme-acmesh
+# webroot option deprecated, use the hardcoded value directly in the next major version
+WEBROOT=${webroot:-/var/run/acme/challenge}
+
+# shellcheck source=net/acme/files/functions.sh
+. /usr/lib/acme/functions.sh
+
+# Needed by acme.sh
+export CURL_CA_BUNDLE=/etc/ssl/certs/ca-certificates.crt
+export NO_TIMESTAMP=1
+
+cmd="$1"
+
+set -- $domains
+main_domain="$1"
+
+case $cmd in
+get)
+	set -- "$ACME"
```

Comment:
updated

Reply:
Missed one `"$@"` invocation :)

Reply:
Sorry, not sure I follow. You mean the first `set` invocation should contain `"$@"`? Since `acme get` current doesn't support any flags, I think we should be defensive here and ignore all flags. The CLI can always be changed latter.

Reply:
No, I meant you missed translating one instance of `"$@"` to `"$ACME" "$@"` (on line 46) :)

Reply:
Ah, nice catch. Fixed.